trivy-image-scan/.trivyignore

```diff
@@ -1,2 +1,2 @@
 # libc6
-CVE-2024-2961 exp:2024-04-30
\ No newline at end of file
+CVE-2024-2961 exp:2024-05-30
```

Comment:
Please don't change global suppressions without verifying if the fix is available (which it now is). Updating base image here: https://github.com/hypertrace/docker-java-base/pull/30

Reply:
Builds were failing on all PRs so made this change as a temp fix 😅